chaoscenter/web/.husky/pre-commit

```diff
@@ -1,8 +1,9 @@
 #!/bin/sh
 "$(dirname "$0")/_/husky.sh"
 
-MATCH_PATH='chaoscenter/web/'
+echo "\033[0;31;1mYou can't commit directly to main branch\033[0m"
```

Comment:
Is this tested properly?
In Litmus there is no main branch, we have a master branch here. Please verify this change.

Reply:
I have not changed the code for this, it is how the original implementation is in `pre-commit.sh`. Should I change it to `master` then?

Reply:
hey @shruti2522 that went unnoticed by me. Please change it to master.